pytest_only/version.py

```diff
@@ -1,6 +1,6 @@
-import pkg_resources
+from importlib import metadata
 
 try:
-    __version__ = pkg_resources.get_distribution('pytest-only').version
+    __version__ = metadata.version('pytest-only')
 except pkg_resources.DistributionNotFound:
```

Comment:
I happened across this because I started getting a deprecation warning for `pkg_resources` when dependencies were updated. Seems like this has little chance of being merged, but figured I'd FYI anyway:

The wrong exception is caught with this change. 
```suggestion
except metadata.PackageNotFoundError
```